python_modules/libraries/dagster-aws/dagster_aws_tests/ecs_tests/launcher_tests/test_launching.py

```diff
@@ -148,90 +235,65 @@ def test_task_definition_registration(
     assert len(ecs.list_task_definitions()["taskDefinitionArns"]) == len(task_definitions) + 1
 
 
-def test_reuse_task_definition(instance):
-    image = "image"
-    secrets = []
+def test_reuse_task_definition(instance, ecs):
```

Comment:
It's a little unclear to me if this change is meant to:

- force registration of a new task definition revision for the same family
- use a task definition without any pre-populated defaults from the parent task

Which probably points to both some poor decisions with how I initially wrote this and also some naming ambiguities that could be cleaned up in this PR?